cli/command/service/opts.go

```diff
@@ -453,6 +453,10 @@ func convertExtraHostsToSwarmHosts(extraHosts []string) []string {
 	return hosts
 }
 
+type scaleOptions struct {
+	detach bool
+}
```

Comment:
minor, but this struct should go in `scale.go` since it's only used in that one file. `serviceOptions` is a rare case where there are a lot of options shared between multiple commands.